docs/user-guide/nginx-configuration/configmap.md

```diff
@@ -102,6 +102,7 @@ The following table shows a configuration option's name, type, and the default v
 |[gzip-level](#gzip-level)|int|1|
 |[gzip-types](#gzip-types)|string|"application/atom+xml application/javascript application/x-javascript application/json application/rss+xml application/vnd.ms-fontobject application/x-font-ttf application/x-web-app-manifest+json application/xhtml+xml application/xml font/opentype image/svg+xml image/x-icon text/css text/javascript text/plain text/x-component"|
 |[worker-processes](#worker-processes)|string|`<Number of CPUs>`|
+|[worker-rlimit-core](#worker-rlimit-core)|string|empty|
```

Comment:
nit here: all other default definitions for string are "" and not the empty word.